docs/cimpl6LanguageReference.md

```diff
@@ -1057,124 +1044,127 @@ With FHIR as the target output, the right-hand side of the map statement can poi
+| `type` |The slices are differentiated by the data type of the discriminator |
+| `profile` | The slices are differentiated by conformance of the discriminator to a specified **profile**. This can only be used when the discriminator is an `Entry`, and is generally not recommended for performance reasons. |
 
 #### Slice Strategy
 <!-- Carmela - confirm underscore with slice strategy check the code -->
 
-The `slice strategy` indicates how the CIMPL `Property` should map to slices in the target FHIR **element**. The default `slice strategy` adds a single slice, corresponding to the CIMPL data type, in the target FHIR array.  To map a CIMPL `Property` with `includes` constraints, such that each included CIMPL data type becomes a slice in the target FHIR array, specify _`slice strategy =` **includes**_.
+The `slice strategy` indicates how the CIMPL `Property` should map to slices in the target FHIR **element**. The default `slice strategy` adds a single slice, corresponding to the CIMPL data type, in the target FHIR array.  To map a CIMPL `Property` with `includes` constraints, such that each included CIMPL data type becomes a slice in the target FHIR array, specify `slice strategy = includes`.
 
-| Reserved Phrase | Example |
-|----------|---------|
-| `slice strategy` | _Components.ObservationComponent `maps to` **component** `(slice on = code.coding.code; slice strategy = includes)`_ |
+#### Slice Location: `slice at`
 
-In the above example, each included _ObservationComponent_ subtype will create a new slice in the FHIR **component** array.  Each slice is distinguished by the value at the path _component.code.coding.code_.
+If the discriminator used for slicing is not the FHIR **element** that is being mapped to, `slice at` allows you to explicitly define the location of the slice.
 
-#### Slice Location (slice at)
+### Slicing Examples
 
-If the discriminator used for slicing is not the FHIR **element** that is being mapped to, `slice at` allows you to explicitly define the location of the slice.
+The best way to understand slicing statements is through examples.
 
-| Reserved Phrase | Example |
-|----------|---------|
-| `slice at` | _Members.Observation `maps to` **related.target** `(slice at = related; slice on = type)`_ |
+#### Example 1
 
-In the above example, though the mapping is declared such that _Members.Observation `maps to` **related.target**_, the slicing is not occurring at **related.target**.  Instead, the _`slice at =` **related**_ ensures that the slicing is occurring at the **_related_ element**.
+`Members.Observation maps to related.target (slice at = related; slice on = type)`
 
-#### Slicing Examples
+In the above example, though the mapping is declared such that _Members.Observation `maps to` **related.target**_, the slicing is not occurring at **related.target**.  Instead, the `slice at = related` ensures that the slicing is occurring at the **related** element.
```

Comment:
Members.Observation maps to related target is a code block - but its italicized here.  `Members.Observation maps to **related.target**`   ?

Reply:
OBE